utils_nlp/eval/rouge_ext.py

```diff
@@ -0,0 +1,978 @@
+# This script is adopted from https://github.com/Diego999/py-rouge/blob/master/rouge/rouge.py
+# to compute ROUGE scores for non-English languages.
+
+# Currently, the script supports Hindi.
+# Additional language support can be added by adding language specific
+# 1) sentence splitter (SENTENCE_SPLIT_DICT)
+# 2) word tokenizer (WORD_TOKENIZE_DICT)
+# 3) pattern of characters to remove (REMOVE_CHAR_PATTERN_DICT)
+# 4) stemmer (STEMMER_DICT), this is optional since stemming is not applicable to all languages
+# 5) changes to _split_into_words if the text words are not separated by space
```

Comment:
_split_into_words still splits words by space, in the current version.

Reply:
Yes. I meant if you want to support languages like Chinese, you need to change this function to handle text that's not separated by space.